modules/account-lib/src/coin/stx/transactionBuilder.ts

```diff
@@ -110,39 +106,41 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
       BufferReader.fromBuffer(Buffer.from(removeHexPrefix(rawTransaction), 'hex')),
     );
     tx.stxTransaction = stackstransaction;
-    const spendingCondition = stackstransaction.auth.spendingCondition;
-    if (spendingCondition) {
-      if (!isSingleSig(spendingCondition)) {
-        this._numberSignatures = spendingCondition.signaturesRequired;
-      } else {
-        this._numberSignatures = 1;
-      }
-    }
     this.initBuilder(tx);
     return this.transaction;
   }
 
   // region Base Builder
   /** @inheritdoc */
   protected async buildImplementation(): Promise<Transaction> {
+    const isMultiSig: boolean = this._fromPubKeys.length > 1;
```

Comment:
why are we continually redefining what multisig criteria are? can we use `!isSingleSig()`here?

Reply:
That condition is relocated from the following reference. It means the opposite of isSingleSig. They are two different things. The plurality of the semantics was changed to avoid confusion. isSingleSig is an internal function of stack.js. On the first try I used isSingleSig. You can go back for that truck if you think it will be clearer.

https://github.com/BitGo/BitGoJS/pull/1421/files#diff-7e28454346122befb99605f628617c5b26224561ccb8411bd2971193c24c91bdL60

Reply:
https://github.com/BitGo/coins-sandbox/blob/8cd53d9c0d69d53b72434a86019ea86a1cae34a1/stack/sendMultiSigMultiStep.js#L18